projects/coordinator/src/main/java/org/batfish/coordinator/Main.java

```diff
@@ -55,6 +58,10 @@ public static PoolMgr getPoolMgr() {
     return _poolManager;
   }
 
+  public static Storage getStorage() {
+    return _storage;
```

Comment:
Suggest to use checkState() function here for consistency.

Reply:
Done